chromadb/utils/system_info_utils.py

```diff
@@ -0,0 +1,103 @@
+import datetime
+import logging
+import os
+import platform
+import psutil
+from typing import Dict, cast, Any
+import re
+import chromadb
+from chromadb.api.types import SystemInfo, ChromaMode
+from chromadb.api import ServerAPI as API
+
+logger = logging.getLogger(__name__)
+
+
+def format_size(size_in_bytes: int) -> str:
+    units = ["B", "KB", "MB", "GB", "TB"]
+    unit_index = 0
+    size = float(size_in_bytes)
+
+    while size > 1024 and unit_index < 4:
+        size /= 1024.0
+        unit_index += 1
+
+    return f"{size:.2f} {units[unit_index]}"
+
+
+SENSITIVE_VARS_PATTERNS = [".*PASSWORD.*", ".*KEY.*", ".*AUTH.*"]
+SENSITIVE_SETTINGS_PATTERNS = [".*credentials.*"]
+
+
+def sanitized_environ() -> Dict[str, str]:
```

Comment:
is this used anywhere? If so can we not? This feels invasive even if there's no actual privacy concern.